light-zk.js/src/test-utils/initLookUpTable.ts

```diff
@@ -175,7 +174,9 @@ export async function initLookUpTableTest(
       lookupTableAddress,
       "confirmed",
     );
-    assert(lookupTableAccount != null);
+    if (!lookupTableAccount) {
+      throw new Error("lookupTableAccount is null or undefined");
```

Comment:
Should we define and import a standardised error here?

Reply:
done, noticed that some errors were not custom in other functions either. Replaced those errors with custom ones as well.